src/Pages/User/AddBlog/AddBlog.jsx

```diff
@@ -5,17 +5,32 @@ import Swal from "sweetalert2";
 import axios from "axios";
 
 const AddBlog = () => {
-    const { register, handleSubmit, reset, formState: { errors }} = useForm();
+    const { register, handleSubmit, reset, formState: { errors } } = useForm();
     const { user } = useContext(AuthContext);
 
-    const defaultName = user ? user.displayName
-        : ""; // Get the default name from the user context
+    const defaultName = user ? user.displayName : ""; // Get the default name from the user context
 
     const onSubmit = (data) => {
+        // Get the current date and time
+        const currentDate = new Date();
+
+        // Get the user's preferred locale
+        const userLocale = navigator.language || "en-US";
+
+        // Format the current date according to the user's locale
+        const formattedDateTime = currentDate.toLocaleDateString(userLocale, {
+            year: "numeric",
+            month: "long",
+            day: "numeric",
+        });
+
```

Comment:
Make a custom date hook so that it can be Reusable. 
Follow DRY(Dont Repeat Yourself) Method